Update PID and EAA (#947)
italia/eidas-it-wallet-docs: docs/en/functionalities.rst

- MUST guarantee that the PID is a functional element, for the User to be authenticated by a Relying Party in a digital context (see :ref:`functionalities:Authentication`), to access services in proximity contexts, and to request the issuance of additional EAAs (see :ref:`functionalities:Issuance of Electronic Attestations of Attributes`);
- MUST display a method of assistance given by the PID Provider (see :ref:`functionalities:User Assistance`); 
- MUST guarantee that the PID is recognizable by the User and distinguishable from other EAAs.  

To ensure a consistent identification and representation of the PID across different Wallet Solutions, the Wallet Provider: 

- MUST use the official naming “IT-Wallet ID” to refer to PID in their Wallet Solutions and MUST NOT use custom or technical terms such as "Person Identification Data" or its acronym "PID"; 
- MUST use the IT-Wallet ID official graphic asset available in the Official Resources and MUST comply with the related usage specifications provided; 

> - MUST use the IT-Wallet ID graphic asset in the application/svg+xml data format; 

Reviewer: ```suggestion
- MUST use the IT-Wallet ID graphic asset in the ``application/svg+xml`` data format; 
```

> - MUST NOT alter, distort, modify, or replace the IT-Wallet ID graphic asset with unofficial graphic assets; 

Reviewer: ```suggestion
- MUST NOT alter, modify, or replace the IT-Wallet ID graphic asset with unofficial graphic assets; 
```

- MUST use the IT-Wallet ID official graphic asset available in the :ref:`official-resources:Official Resources` and MUST comply with the related usage specifications provided; 
- MUST use the IT-Wallet ID graphic asset in the ``application/svg+xml`` data format; 
- MUST NOT alter, modify, or replace the IT-Wallet ID graphic asset with unofficial graphic assets; 
- MUST maintain the minimum clear space as defined in the :ref:`official-resources:Official Resources` to ensure visibility and recognizability. No other graphic or textual elements MUST interfere with this space; 
- MUST NOT resize the IT-Wallet ID graphic asset below the minimum dimensions specified in the :ref:`official-resources:Official Resources` to maintain legibility across formats and devices; 
- MUST NOT place the IT-Wallet ID graphic asset on backgrounds that compromise its visibility or legibility. Adequate contrast between the IT-Wallet ID graphic asset and the background MUST be ensured, in line with the :ref:`official-resources:Official Resources`. 

.. only:: format_html

>   .. figure:: ./images/svg/IT-Wallet-ID.svg

Reviewer: @steps88 both .svg and .pdf are missing

please add them and push the changes

Reviewer: I just have added an automatic check in continuous integration about these issues

Following is an example of informative text: 

.. note::
  **Title:** Who may obtain the document 

>   **Text:** The digital version of [Document name] is available only to those who already hold the physical one. Please, make sure to have already obtained the corresponding physical document.  For more details, [read more information] (URL). 

Reviewer: ```suggestion
  **Text:** The digital version of [Document name] is available only to those who already hold the physical one. Please, make sure to have already obtained the corresponding physical document. For more details, [read more information] (URL). 
```

Focus on Electronic Attestations of Attributes
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

The EAAs obtained within the Wallet Instance MUST ensure a high level of recognizability and accessibility [REF_ACCESSIBILITY] of the information contained. 
The EAAs representation depends on the UI and design choices of the Wallet Solution and for certain specific aspects, on what is defined within the Authentic Source Registry and the Digital Credential Catalog (see :ref:`registry:Registry Infrastructure`).  
Below are the requirements that every Authentic Source MUST follow to personalize their EAAs (see :ref:`registry:Authentic Source Registry`): 

- The EAA MAY be characterized by the **Authentic Source** logo. The Authentic Source MAY provide this logo in two versions, a compact version via the ``organization_info.logo_uri`` parameter and an extended version via the ``organization_info.logo_extended_uri`` parameter. In particular, the Authentic Source:  

>  - MUST provide the logo only in one of the following formats: .png, .svg, or .webp; 

Reviewer: ```suggestion
 - MUST provide the logo only in one of the following formats: ``image/png``, ``image/svg+xml``, or ``image/webp``; 
```

 - MUST provide the logo in both a positive and a negative version, if available; 
 - MUST provide the logo with a minimum size of 30 × 30 pixels and a maximum size of 60 x 60 pixels, in its compact version; 
 - MUST provide the logo with an aspect ratio of 1:1, in its compact version; 
 - MUST provide the logo which does not exceed the maximum file size of 80 KB, in its compact version; 
 - MUST provide the logo with a minimum size of 200 × 30 pixels and a maximum size of 650 × 180 pixels, in its extended version; 
 - MUST provide the logo which does not exceed the maximum file size of 150 KB, in its extended version. 

- The EAA MAY be characterized by a **distinctive logo**. The Authentic Source MAY provide this logo via the ``data_capabilities.logo_uri`` parameter. In particular, the Authentic Source: 

>  - MUST provide the logo only in one of the following formats: .png, .svg, or .webp; 

Reviewer: ```suggestion
 - MUST provide the logo only in one of the following formats: ``image/png``, ``image/svg+xml``, or ``image/webp``; 
```

 - MUST provide the logo in both a positive and a negative version, if available; 
 - MUST provide the logo with a minimum size of 200 × 30 pixels and a maximum size of 650 × 180 pixels; 
 - MUST provide the logo with a maximum file size of 150 KB. 

- The EAA MAY be characterized by a background image. The Authentic Source MAY provide the desired image via the ``data_capabilities.background_image`` parameter. In particular, the Authentic Source: 

 - MUST provide the background image as a single image; 

>  - MUST provide the background image only in one of the following formats: .png, .svg, .jpeg, .jpg, .webp; 

Reviewer: ```suggestion
 - MUST provide the background image only in one of the following formats: ``image/png``, ``image/svg+xml``, ``image/jpeg`` or ``image/webp``; 
```

 - MUST provide the background image with a minimum resolution of 321 × 205 pixels and maximum resolution of 1284 × 820 pixels; 
 - MUST provide the background image with a maximum file size of 200 KB. 

- The EAA MAY be characterized by a **background watermark**. The Authentic Source MAY provide the desired watermark via the ``data_capabilities.watermark_image`` parameter. In particular, the Authentic Source: 
 
 - MUST provide the watermark as a single element; 

>  - MUST provide the watermark only in one of the following formats: .png, .svg; 

Reviewer: ```suggestion
 - MUST provide the watermark only in one of the following formats: ``image/png``, ``image/svg+xml``; 
```

 

**"Login with IT-Wallet" button – svg**

> For further information on the Authentication button refer to the Brand Manual which will soon by made available on the official website https://www.wallet.gov.it. Authentication button Official Resource is available in the related Official Resources folder of these Technical Specifications. 

Reviewer: https://www.wallet.gov.it should be moved in the official resources and an HREF to it must be used

all these occurency will be therefore replaced with the HREF

Reviewer: https://github.com/italia/eid-wallet-it-docs/issues/954
